lms/templates/discovery/facet.underscore

```diff
@@ -1,16 +1,16 @@
 <h3 class="header-facet">
-    <%= displayName %>
+    <%- displayName %>
 </h3>
-<ul data-facet="<%= name %>" class="facet-list collapse">
-    <%= options %>
+<ul data-facet="<%- name %>" class="facet-list collapse">
+    <%= HtmlUtils.ensureHtml(optionsHtml) %>
```

Comment:
This should not be escaped in the template because it is HTML (and it has been marked as such).